service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsController.java

```diff
@@ -80,13 +85,15 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
         CaseDetails caseDetails = request.getCaseDetails();
         CaseData caseData = getCaseData(caseDetails);
         ManageDocumentType type = caseData.getManageDocument().getType();
+        boolean isSolicitor = userService.hasAnyCaseRoleFrom(representativeSolicitors(), caseData.getId().toString());
```

Comment:
maybe throw error if user is HMCTS and has a solicitor role, this indicates their account is essentially broken